cwms/timeseries/timeseries_bin.py

```diff
@@ -2,20 +2,20 @@
 #  United States Army Corps of Engineers - Hydrologic Engineering Center (USACE/HEC)
 #  All Rights Reserved.  USACE PROPRIETARY/CONFIDENTIAL.
 #  Source may not be released without written approval from HEC
-import datetime
 import json
+from datetime import datetime
 from enum import Enum, auto
+from typing import Optional
 
 import cwms._constants as constants
-from cwms.core import CwmsApiSession
-from cwms.core import _CwmsBase
-from cwms.utils import queryCDA
-from cwms.utils import raise_for_status
+from cwms.core import CwmsApiSession, _CwmsBase
+from cwms.types import JSON
+from cwms.utils import queryCDA, raise_for_status
 
 
 class DeleteMethod(Enum):
-    DELETE_ALL = auto(),
-    DELETE_KEY = auto(),
+    DELETE_ALL = (auto(),)
```

Comment:
Why are these two different than the last one?

Reply:
I think it's because the first two enum fields are defined as tuples (due to the trailing comma).

Reply:
I agree, but that's how they are defined in the `main` branch. This pull request only deals with formatting and type hints, and in this case the code is formatted correctly, given the input.

Reply:
Do you want me to do a follow-up PR fixing that?

Reply:
That would be great, thanks. I'd prefer to keep this pull request limited to formatting/type fixes (especially since most of the changes were generated automatically). Feel free to submit a PR once this one is approved/merged.